### ISSUE TITLE
Python3: import of twisted.conch.ssh.keys fails

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8220]
|<img alt="doko's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| doko reported|
|-|-|
|Trac ID|trac#8220|
|Type|release blocker: wrong release notes|
|Created|2016-02-25 16:38:23Z|
|Branch|https://github.com/twisted/twisted/tree/missing-dist3-module-8220-2|

$ python3 -c 'import twisted.conch.ssh.keys'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/twisted/conch/ssh/keys.py", line 34, in <module>
    from twisted.conch.ssh import common, sexpy
  File "/usr/lib/python3/dist-packages/twisted/conch/ssh/common.py", line 14, in <module>
    from twisted.conch.ssh._cryptography_backports import (
ImportError: No module named 'twisted.conch.ssh._cryptography_backports'

Attachments:

* [issue8220.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/60e/60e6fccfcf01252eea28e046a86487e4bce4685c/c5fcad0a74244a0c0999dba818b84b283bd00994.diff) (416 bytes) - added by doko on 2016-02-25 16:44:23Z - patch

<details><summary>Searchable metadata</summary>

```
trac-id__8220 8220
type__release_blocker__wrong_release_notes release blocker: wrong release notes
reporter__doko doko
priority__normal normal
milestone__Twisted_16_0 Twisted 16.0
branch__branches_missing_dist3_module_8220_2 branches/missing-dist3-module-8220-2
branch_author__mithrandi mithrandi
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1456418303867747 1456418303867747
changetime__1456631539253392 1456631539253392
version__None None
owner__mithrandi mithrandi

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

should be block the release on this issue?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

doko: thanks for the report, and the patch.

adi: It's not _actually_ a regression, as conch was not present in 15.5; but I am going to leave that tag on there, because it makes the release notes inaccurate.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

added a new tag for the relevant ticket type

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_4"></a>|@mithrandi commented|
|-|-|

(In [46853]) Branching to missing-dist3-module-8220.

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_5"></a>|@mithrandi commented|
|-|-|

(In [46856]) Branching to missing-dist3-module-8220-2.

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_6"></a>|@mithrandi commented|
|-|-|

In the process of trying to verify this fix, I ran into [#8221](https://github.com/twisted/twisted/issues/8221), and I'm stuck on that.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set owner to @mithrandi|
|-|-|

Thanks mithrandi!

- Please add a topfile (probably a .misc).
- Please add ```from __future__ import absolute_import, division``` into the backports file.

Once you've fixed these two issues, and the builders are all green, please merge. Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_9"></a>|@mithrandi set status to `closed`|
|-|-|

(In [46866]) Merge missing-dist3-module-8220-2: Add a missing module to twisted.python.dist3

Fixes: [#8220](https://github.com/twisted/twisted/issues/8220)
Author: doko
Reviewer: hawkowl
